docs/Configure-Azure-Deploy.md

```diff
@@ -0,0 +1,77 @@
+# Introduction
+
+Tutorial covers configuration of Admin for deploy on Azure.
+
+
+## Create database
+
+If you don't have publicly accessible database you will need to create one. Follow tutorials for creating databases on Azure:
+
+- [SQL Server](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-get-started-portal)
+- [PostgreSQL](https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-server-database-portal)
+- [MySQL](https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-portal)
+
+Replace connection strings in appsettings with connection string to generated databse.
+
+Then you can generate migrations and update the database. In `src/Skoruba.IdentityServer4.Admin` issue:
+
+```
+dotnet ef migrations add DbInit -c AdminDbContext -o Data/Migrations
+dotnet ef database update -c AdminDbContext
+```
+
+## Deploying webbaps to Azure App Service
+
+We will assume in the tutorial that STS and Admin were deployed to:
+
+- https://is4-sts.azurewebsites.net - STS
+- https://is4-admin.azurewebsites.net - Admin panel
+
+
+### Updating urls
+
+Remember to replace those values with your own in `src/Skoruba.IdentityServer4.Admin/appsettings.json` before first publish -
+
+```json
+"AdminConfiguration": {
+	"IdentityAdminBaseUrl": "https://is4-admin.azurewebsites.net",
+	"IdentityAdminRedirectUri": "https://is4-admin.azurewebsites.net/signin-oidc",
+	"IdentityServerBaseUrl": "https://is4-sts.azurewebsites.net"
+}
+```
+
+Then follow instructions from [Quickstart deploy to Azure](https://docs.microsoft.com/en-us/visualstudio/deployment/quickstart-deploy-to-azure) 
+
+
+### Adding certificate for signing tokens
+
+We also need to upload pfx certificate for signing tokens. If you don't have one here are the steps to do it using openssl (remember to write down the password - we will need it later):
```

Comment:
It would be good to link OpenSSL installation instructions on Windows. Or an alternative Windows OOTB way of generating these certificates.